<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="fromUserbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QComboBox" name="toUserbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="calBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Calendar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remove">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="editor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reviewer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Reviewer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="owner">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Owner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
